cleanup-passes/v3-elements.ts

```diff
@@ -0,0 +1,172 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.  This
+ * code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+'use strict';
+
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import * as yaml from 'js-yaml';
+import {execFile} from 'child_process';
+import {promisify} from 'util';
+
+import {register} from '../cleanup-pass';
+import {ElementRepo} from '../element-repo';
+import {makeCommit} from './util';
+
+const execFilePromise = promisify(execFile);
+
+const filesWeMightChange = new Set<string>([
+  '.travis.yml',
+  '.npmignore',
+  '.gitignore',
+  'package.json',
+  'package-lock.json',
+]);
+
+/**
+ * Perform updates for 3.0 Polymer elements.
+ */
+async function v3ElementPass(element: ElementRepo): Promise<void> {
+  await updateNpmIgnore(element);
+  await updateGitIgnore(element);
+  await updateTravisYaml(element);
+  await updatePackageJson(element);
+  await regeneratePackageLock(element);
+
+  let shouldCommit = false;
+  for (const changedFile of await element.repo.getStatus()) {
```

Comment:
This loop is doing two things, checking to that changedFiles isn't empty, and asserting that it only includes expected files. Recommend pulling `await element.repo.getStatus()` and doing these two operations separately for clarity.